src/entities/Proposal/model.ts

```diff
@@ -358,13 +350,13 @@ export default class ProposalModel extends Model<ProposalAttributes> {
 
     const timeFrame = this.parseTimeframe(filter.timeFrame)
     const timeFrameKey = filter.timeFrameKey || 'created_at'
-    const orderDirection = order || 'DESC'
+    const orderDirection = !order ? SortingOrder.DESC : order
```

Comment:
I think it's the same to write `const orderDirection = order || SortingOrder.DESC` 🤔